src/utils/progressBar.ts

```diff
@@ -9,9 +9,16 @@ import * as util from 'util';
 import cli from 'cli-ux';
 import { env, once } from '@salesforce/kit';
 import { MetadataApiDeploy } from '@sf/sdr';
+import { Messages } from '@salesforce/core';
 
-const startProgressBar = once((bar: ProgressBar.Bar, total: number) => {
+Messages.importMessagesDirectory(__dirname);
```

Comment:
adjusting local functions to handle progress bar that displays deploy status as part of progress bar.